en/examples/follow_me.md

```diff
@@ -0,0 +1,341 @@
+# Example: Follow Me Mode
+
+This example demonstrates how to use the [Follow Me](../api_reference/classdronecore_1_1_follow_me.md) plugin. 
+It shows how to send the drone both the current position of the target (`FollowMe::TargetLocation`) and the relative position at which it should follow (`FollowMe::Config`).
+
+![Follow Me QGC Screenshot](../../assets/examples/follow_me/follow_me_example_qgc.jpg)
+
+> **Note** A real application using this API will get the position information from the underlying device. The example uses a fake position source (`FakeLocationProvider`) to enable it to be run on computers that do not have position information. The `FakeLocationProvider` emulates the typical usage of common positioning APIs used in Android, Linux and iPhone.
+
+## Running the Example {#run_example}
+
+Before running this example you will need to install Boost libraries. For Linux this is done as shown below:
+```sh
+sudo apt-get install libboost-all-dev
+```
+
+Otherwise example is built and run and run in the normal way ([as described here](../examples/README.md#trying_the_examples)). 
+
+The example terminal output should be similar to that shown below:
+
+> **Note** This is from a debug build of DroneCore. A release build will omit the "Debug" messages.
+
+```
+$ ./follow_me 
+...
+```
+
+## How it works
+
+The example registers with `FakeLocationProvider` for location updates. 
+These are passed to the Follow Me plugin, which in turn sends them to the vehicle. 
+
+The operation of the "DroneCore" part of this code is discussed in the guide: [Follow Me](../guide/follow_me.md).
+
+
+## Source code {#source_code}
+
+> **Tip** The full source code for the example [can be found on Github here](https://github.com/dronecore/DroneCore/tree/{{ book.github_branch }}/example/fly_mission).
+
+
+[CMakeLists.txt](https://github.com/dronecore/DroneCore/blob/{{ book.github_branch }}/example/follow_me/CMakeLists.txt)
+
+```make
+cmake_minimum_required(VERSION 2.8.12)
+
+project(follow_me)
+
+find_package(Boost 1.66 COMPONENTS REQUIRED system)
+include_directories(${Boost_INCLUDE_DIR})
+
+if(NOT MSVC)
+    add_definitions("-std=c++11 -Wall -Wextra -Werror")
+else()
+    add_definitions("-std=c++11 -WX -W2")
+    include_directories(${CMAKE_SOURCE_DIR}/../../install/include)
+    link_directories(${CMAKE_SOURCE_DIR}/../../install/lib)
+endif()
+
+add_executable(follow_me
+    follow_me.cpp
+    fake_location_provider.cpp
+)
+
+target_link_libraries(follow_me
+    LINK_PUBLIC ${Boost_LIBRARIES}
+    dronecore
+    dronecore_action
+    dronecore_follow_me
+    pthread
```

Comment:
Do we really need `pthread` here? Doesn't dronecore link against that?

Reply:
Yes, its not necessary anymore. Fixed here https://github.com/dronecore/DroneCore/pull/235/commits/91e0a2d325225f8497fd3c4116420a4867f8de18. Thanks @hamishwillee

Reply:
Thanks! Corresponding change in docs here: https://github.com/dronecore/docs/commit/a658b68d7362f51e39cf61acb5806fe6af1c408f